app/src/main/java/com/example/cocktails/ScrollingActivity.kt

```diff
@@ -42,6 +38,8 @@ class ScrollingActivity : AppCompatActivity() {
 
     private var gridViewAdapter: CocktailItemAdapter? = null
     private lateinit var recyclerView: RecyclerView
+    private lateinit var addUserItemFab: View
```

Comment:
@enavhasson why saving it? you are using it only once via initAddUserItemButton, am I right?